tools/sctool/src/main/java/com/horizen/ScBootstrappingToolCommandProcessor.java

```diff
@@ -66,6 +68,9 @@ public void processCommand(String input) throws IOException {
             case "generateCswProofInfo":
                 processGenerateCswProofInfo(command.data());
                 break;
+            case "encodeString":
```

Comment:
do we really need this change to the bootstrapping tool?
looks like it just performs a standar Blake2b256, which should be something standard and could be generated by some online tools as well

Reply:
Agree but i think we should give the possibility to the user to calculate it using also our tools.

Reply:
I would prefer to keep it as out-of-the-box utility.
Need to update the internal doc with the usage info.

Reply:
Which doc are you referring to @i-Alex ?

Reply:
Updated also the README with the ScBoostrapping tool instruction.

Reply:
good